### DISCUSSION TITLE
ScriptReference/Rendering.CommandBuffer.SetRenderTarget

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#417] [Notes]
# ScriptReference/Rendering.CommandBuffer.SetRenderTarget

https://docs.unity3d.com/ScriptReference/Rendering.CommandBuffer.SetRenderTarget.html

**Comment by ghost** [comment]
Important note, when targeting Texture3D or Texture2DArrays, if you want to use `uint slice : SV_RenderTargetArrayIndex;` for writing into multiple slices within the geometry shader, you must use `_cmd.SetRenderTarget(targetName, 0, CubemapFace.Unknown, -1);`
